README.adoc

```diff
@@ -914,13 +915,13 @@ Always use https://github.com/travisjeffery/timecop[Timecop] instead of stubbing
 it 'offsets the time 2 days into the future' do
   current_time = Time.now
   allow(Time).to receive(:now).and_return(current_time)
-  expect(subject.get_offset_time).to eq(current_time + 2.days)
+  expect(subject).to eq(current_time + 2.days)
```

Comment:
I struggle to understand (both in the old and in the new examples) what the subject is.

Reply:
I considered adding a `describe` block around these examples:

```ruby
# Has a method #get_offset_time, which returns the current time offset 2 days into the future
describe TimeShifter do
  # bad
  describe '#get_offset_time' do
    subject { TimeShifter.new.get_offset_time }

    it 'offsets the time 2 days into the future' do
      current_time = Time.now
      allow(Time).to receive(:now).and_return(current_time)
      expect(subject).to eq(current_time + 2.days)
    end
  end

  # good
  describe '#get_offset_time' do
    subject { TimeShifter.new.get_offset_time }

    it 'offsets the time 2 days into the future' do
      Timecop.freeze(Time.now) do
        expect(subject).to eq 2.days.from_now
      end
    end
  end
end
```

I don't think the function of the class or method under test is a huge concern here - all it's really trying to demonstrate is that we're dealing with time, but I can see why the lack of context would lead to more confusion. Is that context a good improvement?

Reply:
Also, I guess it's worth not using `from_now` in the test, since the implemented functionality would probably use that too. `Timecop.freeze('Mon, 24 Jan 2022 11:47:04 +0000') { expect(subject).to eq 'Mon, 24 Jan 2022 13:47:04 +0000' }` would probably be better (if you coerce the strings to datetimes).

Reply:
What the bad and the good examples try to show is not a spec for a time shifter, but rather a spec for a time sensitive class where time shifting comes handy. Think `InvoiceReminder`.